service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/kubernetesNamespace/ControlledAzureKubernetesNamespaceResource.java

```diff
@@ -280,12 +281,6 @@ private List<Step> getFederatedCredentialsDeleteSteps(FlightBeanBag flightBeanBa
     if (requiresFederatedCredentials()) {
       return List.of(
           getGetManagedIdentityStep(flightBeanBag),
-          new GetFederatedIdentityStep(
```

Comment:
Why is this step not necessary?

Reply:
let me know if this does not cover it https://github.com/DataBiosphere/terra-workspace-manager/pull/1488#discussion_r1337764107